modules/uid20IdSystem.js

```diff
@@ -0,0 +1,97 @@
+/**
+ * This module adds uid2 ID support to the User ID module
+ * The {@link module:modules/userId} module is required.
+ * @module modules/uid2IdSystem
+ * @requires module:modules/userId
+ */
+
+import * as utils from '../src/utils.js'
+import {submodule} from '../src/hook.js';
+import {getStorageManager} from '../src/storageManager.js';
+
+const storage = getStorageManager();
+
+const MODULE_NAME = 'uid20';
+const GVLID = 887;
+const LOG_PRE_FIX = 'UID20: ';
+const ADVERTISING_COOKIE = '__uid2_advertising_token';
+
+const logInfo = createLogInfo(LOG_PRE_FIX);
+
+function createLogInfo(prefix) {
+  return function (...strings) {
+    utils.logInfo(prefix + ' ', ...strings);
+  }
+}
+
+/**
+ * Encode the id
+ * @param value
+ * @returns {string|*}
+ */
+function encodeId(value) {
+  const result = {};
+  if (value) {
+    const bidIds = {
+      id: value
+    }
```

Comment:
what's the purpose of setting this `const` when it's just used once and immediately on the line below?